### PR TITLE
zh-trans: website/content/zh/docs/user-journeys/users/cluster-operato…

### PR DESCRIPTION
**Pull request by zhangx501**
…r/intermediate.md

>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Remember to delete this note before submitting your pull request.
>
> For pull requests on 1.15 Features: set Milestone to 1.15 and Base Branch to dev-1.15
> 
> For pull requests on Chinese localization, set Base Branch to release-1.14
>
> For pull requests on Korean Localization: set Base Branch to dev-1.14-ko.\<latest team milestone>
>
> If you need Help on editing and submitting pull requests, visit:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> If you need Help on choosing which branch to use, visit:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

**Comment by zhangx501** [comment]
assign @dchen1107
Please give this PR approval.

File: content/zh/docs/user-journeys/users/cluster-operator/intermediate.md [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+{{% capture body %}}
+<!--
+## Work with ingress, networking, storage, and workloads
+-->
+## 使用入口，网络，存储和工作负载
+
+<!--
+Introductions to Kubernetes typically discuss simple stateless applications. As you move into more complex development, testing, and production environments, you need to consider more complex cases:
+-->
+Kubernetes的介绍通常讨论简单的无状态应用程序。当您进入更复杂的开发，测试和生产环境时，您需要考虑更复杂的情况：
```
Comment (xiangpengzhao) [inline content/zh/docs/user-journeys/users/cluster-operator/intermediate.md]: Kubernetes 后面增加空格

File: content/zh/docs/user-journeys/users/cluster-operator/intermediate.md [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+* [Managing Compute Resources for Containers](/docs/concepts/configuration/manage-compute-resources-container/)
+* [Configuration Best Practices](/docs/concepts/configuration/overview/)
+-->
+以及 Pod 如何处理调度，优先级，中断：
+
+* [Taints 和 Tolerations](/docs/concepts/configuration/taint-and-toleration/)
+* [Pod 和优先级](/docs/concepts/configuration/pod-priority-preemption/)
+* [中断](/docs/concepts/workloads/pods/disruptions/)
+* [分配 Pod 到 Node](/docs/concepts/configuration/assign-pod-node/)
+* [管理容器的计算机资源](/docs/concepts/configuration/manage-compute-resources-container/)
```
Comment (xiangpengzhao) [inline content/zh/docs/user-journeys/users/cluster-operator/intermediate.md]: `计算机`-->`计算`

File: content/zh/docs/user-journeys/users/cluster-operator/intermediate.md [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+* [Using RBAC Authorization](/docs/reference/access-authn-authz/rbac/)
+
+You should create Secrets to hold sensitive data such as passwords, tokens, or keys. Be aware, however, that there are limitations to the protections that a Secret can provide. See [the Risks section of the Secrets documentation](/docs/concepts/configuration/secret/#risks).
+-->
+您还可以配置授权。也就是说，您不仅可以决定用户和服务如何向 API server 进行身份认证，或者确定他们是否可以访问，还可以决定他们有权访问哪些资源。基于角色的访问控制（RBAC）是控制对 Kubernetes 资源授权的推荐机制。其他授权模式可以用于更具体的使用场景。
+
+* [授权概述](/docs/reference/access-authn-authz/authorization/)
+* [使用 RBAC 授权](/docs/reference/access-authn-authz/rbac/)
+
+您应该创建 Secrets 来保存敏感数据，例如密码，令牌或者密钥。但请注意，Secret可以提供的保护存在限制。请参阅 [Secrets 文档中的风险部分](/docs/concepts/configuration/secret/#risks)。
```
Comment (xiangpengzhao) [inline content/zh/docs/user-journeys/users/cluster-operator/intermediate.md]: Secret 后面增加空格

File: content/zh/docs/user-journeys/users/cluster-operator/intermediate.md [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+
+<!--
+## Implement custom logging and monitoring
+-->
+## 实现自定义日志和监控
+
+<!--
+Monitoring the health and state of your cluster is important. Collecting metrics, logging, and providing access to that information are common needs. Kubernetes provides some basic logging structure, and you may want to use additional tools to help aggregate and analyze log data.
+-->
+监控集群的健康和状态非常重要。收集指标，记录，并提供对这些信息的访问是常见的需求。Kubernetes 提供了一些基本的日志记录结构，您可能需要使用其他工具来帮助汇总和分析日志数据。
```
Comment (xiangpengzhao) [inline content/zh/docs/user-journeys/users/cluster-operator/intermediate.md]: `指标，记录`最好改为`指标、日志`

File: content/zh/docs/user-journeys/users/cluster-operator/intermediate.md [hunk 5]
```diff
@@ -0,0 +1,221 @@
...
+-->
+如果您是一个集群操作者，希望扩展对 Kubernetes 的理解，本页及其相关主题会扩展[基础集群操作者页面](/docs/user-journeys/users/cluster-operator/foundational)上提供的信息。在本页中，你可以获得如何管理一个完整生产集群的关键 Kubernetes 任务的信息。
+
+{{% /capture %}}
+
+{{% capture body %}}
+<!--
+## Work with ingress, networking, storage, and workloads
+-->
+## 使用入口，网络，存储和工作负载
```
Comment (chenrui333) [inline content/zh/docs/user-journeys/users/cluster-operator/intermediate.md]: ingress 这个 入口翻译不太好。直接用原文吧

File: content/zh/docs/user-journeys/users/cluster-operator/intermediate.md [hunk 6]
```diff
@@ -0,0 +1,221 @@
...
+* [Taints and Tolerations](/docs/concepts/configuration/taint-and-toleration/)
+* [Pods and Priority](/docs/concepts/configuration/pod-priority-preemption/)
+* [Disruptions](/docs/concepts/workloads/pods/disruptions/)
+* [Assigning Pods to Nodes](/docs/concepts/configuration/assign-pod-node/)
+* [Managing Compute Resources for Containers](/docs/concepts/configuration/manage-compute-resources-container/)
+* [Configuration Best Practices](/docs/concepts/configuration/overview/)
+-->
+以及 Pod 如何处理调度，优先级，中断：
+
+* [Taints 和 Tolerations](/docs/concepts/configuration/taint-and-toleration/)
```
Comment (chenrui333) [inline content/zh/docs/user-journeys/users/cluster-operator/intermediate.md]: https://github.com/k8smeetup/k8s-official-translation/blob/master/trans-glossary.md

```suggestion
* [污点 和 容忍度](/docs/concepts/configuration/taint-and-toleration/)
```

File: content/zh/docs/user-journeys/users/cluster-operator/intermediate.md [hunk 7]
```diff
@@ -0,0 +1,221 @@
...
+* [Pod 和优先级](/docs/concepts/configuration/pod-priority-preemption/)
+* [中断](/docs/concepts/workloads/pods/disruptions/)
+* [分配 Pod 到 Node](/docs/concepts/configuration/assign-pod-node/)
```
Comment (chenrui333) [inline content/zh/docs/user-journeys/users/cluster-operator/intermediate.md]:
```suggestion
* [分配 Pod 到工作节点](/docs/concepts/configuration/assign-pod-node/)
```

File: content/zh/docs/user-journeys/users/cluster-operator/intermediate.md [hunk 8]
```diff
@@ -0,0 +1,221 @@
...
+
+<!--
+You also configure authorization. That is, you determine not just how users and services authenticate to the API server, or whether they have access, but also what resources they have access to. Role-based access control (RBAC) is the recommended mechanism for controlling authorization to Kubernetes resources. Other authorization modes are available for more specific use cases.
+
+* [Authorization Overview](/docs/reference/access-authn-authz/authorization/)
+* [Using RBAC Authorization](/docs/reference/access-authn-authz/rbac/)
+
+You should create Secrets to hold sensitive data such as passwords, tokens, or keys. Be aware, however, that there are limitations to the protections that a Secret can provide. See [the Risks section of the Secrets documentation](/docs/concepts/configuration/secret/#risks).
+-->
+您还可以配置授权。也就是说，您不仅可以决定用户和服务如何向 API server 进行身份认证，或者确定他们是否可以访问，还可以决定他们有权访问哪些资源。基于角色的访问控制（RBAC）是控制对 Kubernetes 资源授权的推荐机制。其他授权模式可以用于更具体的使用场景。
```
Comment (chenrui333) [inline content/zh/docs/user-journeys/users/cluster-operator/intermediate.md]:
```suggestion
您还可以配置授权。也就是说，您不仅可以决定用户和服务如何向 API 服务器进行身份认证，或者确定他们是否可以访问，还可以决定他们有权访问哪些资源。基于角色的访问控制（RBAC）是控制对 Kubernetes 资源授权的推荐机制。其他授权模式可以用于更具体的使用场景。
```

File: content/zh/docs/user-journeys/users/cluster-operator/intermediate.md [hunk 9]
```diff
@@ -0,0 +1,221 @@
...
+从 [Kubernetes 日志基础知识](/docs/concepts/cluster-administration/logging/) 开始，了解容器如何执行日志记录和常见模式。集群运维人员通常希望添加一些东西来收集和聚合这些日志。请参阅以下主题：
+
+* [使用 Elasticsearch 和 Kibana 记录日志](/docs/tasks/debug-application-cluster/logging-elasticsearch-kibana/)
+* [使用 Stackdriver 记录日志](/docs/tasks/debug-application-cluster/logging-stackdriver/)
+
+<!--
+Like log aggregation, many clusters utilize additional software to help capture metrics and display them. There is an overview of tools at [Tools for Monitoring Compute, Storage, and Network Resources](/docs/tasks/debug-application-cluster/resource-usage-monitoring/).
+Kubernetes also supports a [resource metrics pipeline](/docs/tasks/debug-application-cluster/resource-metrics-pipeline/) which can be used by Horizontal Pod Autoscaler with custom metrics.
+-->
+与日志聚合一样，许多集群使用其他软件来帮助捕获 metrics 并显示它们。这里是相关工具的概述：[用于监视计算，存储和网络资源的工具](/docs/tasks/debug-application-cluster/resource-usage-monitoring/)。Kubernetes 还支持[资源 metrics 管道](/docs/tasks/debug-application-cluster/resource-metrics-pipeline/)，还可以在水平 Pod 自动伸缩器（Horizontal Pod Autoscaler）中使用自定义 metrics。
```
Comment (chenrui333) [inline content/zh/docs/user-journeys/users/cluster-operator/intermediate.md]:
```suggestion
与日志聚合一样，许多集群使用其他软件来帮助捕获 metrics 并显示它们。这里是相关工具的概述：[用于监视计算，存储和网络资源的工具](/docs/tasks/debug-application-cluster/resource-usage-monitoring/)。Kubernetes 还支持[资源指标管道](/docs/tasks/debug-application-cluster/resource-metrics-pipeline/)，还可以在水平 Pod 自动伸缩器（Horizontal Pod Autoscaler）中使用自定义指标。
```

File: content/zh/docs/user-journeys/users/cluster-operator/intermediate.md [hunk 10]
```diff
@@ -0,0 +1,221 @@
...
+
+<!--
+[Prometheus](https://prometheus.io/), which is another CNCF project, is a common choice to support capture and temporary collection of metrics. There are several options for installing Prometheus, including using the [stable/prometheus](https://github.com/kubernetes/charts/tree/master/stable/prometheus) [helm](https://helm.sh/) chart, and CoreOS provides a [prometheus operator](https://github.com/coreos/prometheus-operator) and [kube-prometheus](https://github.com/coreos/prometheus-operator/tree/master/contrib/kube-prometheus), which adds on Grafana dashboards and common configurations.
+-->
+[Prometheus](https://prometheus.io/)是另一个 CNCF 项目，它是支持捕获和临时收集 metrics 的常用选择。安装 Prometheus 有几种方式，包括使用[stable/prometheus](https://github.com/kubernetes/charts/tree/master/stable/prometheus) [helm](https://helm.sh/) 图表，CoreOS 提供的[prometheus operator](https://github.com/coreos/prometheus-operator) 和 [kube-prometheus](https://github.com/coreos/prometheus-operator/tree/master/contrib/kube-prometheus)，增加了 Grafana 仪表板和常用配置。
```
Comment (chenrui333) [inline content/zh/docs/user-journeys/users/cluster-operator/intermediate.md]: 我看 helm chart 好像也不太好翻译，https://whmzsu.github.io/helm-doc-zh-cn/
```suggestion
[Prometheus](https://prometheus.io/)是另一个 CNCF 项目，它是支持捕获和临时收集指标的常用选择。安装 Prometheus 有几种方式，包括使用[stable/prometheus](https://github.com/kubernetes/charts/tree/master/stable/prometheus) [helm](https://helm.sh/) chart，CoreOS 提供的[prometheus operator](https://github.com/coreos/prometheus-operator) 和 [kube-prometheus](https://github.com/coreos/prometheus-operator/tree/master/contrib/kube-prometheus)，增加了 Grafana 仪表板和常用配置。
```

File: content/zh/docs/user-journeys/users/cluster-operator/intermediate.md [hunk 11]
```diff
@@ -0,0 +1,221 @@
...
+There is a [walkthrough of how to install this configuration in your cluster](https://blog.kublr.com/how-to-utilize-the-heapster-influxdb-grafana-stack-in-kubernetes-for-monitoring-pods-4a553f4d36c9).
+As of Kubernetes 1.11, Heapster is deprecated, as per [sig-instrumentation](https://github.com/kubernetes/community/tree/master/sig-instrumentation).  See [Prometheus vs. Heapster vs. Kubernetes Metrics APIs](https://brancz.com/2018/01/05/prometheus-vs-heapster-vs-kubernetes-metrics-apis/) for more information alternatives.
+-->
+[Minikube](https://github.com/kubernetes/minikube) 和某些 Kubernetes 集群的通常配置是[与  InfluxDB 和 Grafana 一起](https://github.com/kubernetes/heapster/blob/master/docs/influxdb.md)使用[Heapster](https://github.com/kubernetes/heapster)。这里有一个[如何在集群中安装这个配置的参考](https://blog.kublr.com/how-to-utilize-the-heapster-influxdb-grafana-stack-in-kubernetes-for-monitoring-pods-4a553f4d36c9)。截止 Kubernetes 1.11，根据[sig-instrumentation](https://github.com/kubernetes/community/tree/master/sig-instrumentation)，Heapster 已经弃用。更多信息，请参阅 [Prometheus vs. Heapster vs. Kubernetes Metrics APIs](https://brancz.com/2018/01/05/prometheus-vs-heapster-vs-kubernetes-metrics-apis/) 。
```
Comment (chenrui333) [inline content/zh/docs/user-journeys/users/cluster-operator/intermediate.md]:
```suggestion
[Minikube](https://github.com/kubernetes/minikube) 和某些 Kubernetes 集群的通常配置是[与 InfluxDB 和 Grafana 一起](https://github.com/kubernetes/heapster/blob/master/docs/influxdb.md)使用 [Heapster](https://github.com/kubernetes/heapster)。这里有一个[如何在集群中安装这个配置的参考](https://blog.kublr.com/how-to-utilize-the-heapster-influxdb-grafana-stack-in-kubernetes-for-monitoring-pods-4a553f4d36c9)。截止 Kubernetes 1.11，根据 [sig-instrumentation](https://github.com/kubernetes/community/tree/master/sig-instrumentation)，Heapster 已经弃用。更多信息，请参阅 [Prometheus vs. Heapster vs. Kubernetes Metrics APIs](https://brancz.com/2018/01/05/prometheus-vs-heapster-vs-kubernetes-metrics-apis/) 。
```

File: content/zh/docs/user-journeys/users/cluster-operator/intermediate.md (hunk 10 above)
Comment (zhangx501) [reply content/zh/docs/user-journeys/users/cluster-operator/intermediate.md]: 我把 helm chart 保留，这样好像比较好

Comment (zhangx501) [reply content/zh/docs/user-journeys/users/cluster-operator/intermediate.md]: @chenrui333

**Pull request merged**